packages/react-badge/src/Badge/useBadge.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { BadgeProps, BadgeState } from './Badge.types';
+
+export const badgeShorthandProps = ['icon'];
+
+const mergeProps = makeMergeProps<BadgeState>({ deepMerge: badgeShorthandProps });
+
+export const useBadge = (props: BadgeProps, ref: React.Ref<HTMLElement>, defaultProps?: BadgeProps): BadgeState => {
```

Comment:
You should run `create-component` and copy the the comments to relevant files. It's important for things like docgen (componentInfo) and conformance tests

Reply:
Done